data/json/furniture_and_terrain/furniture-graves.json

```diff
@@ -95,5 +95,35 @@
       "sound_fail": "thunk!",
       "items": [ { "item": "rock", "count": [ 18, 30 ] } ]
     }
+  },
+  {
+    "type": "furniture",
+    "id": "f_battlefield_cross",
+    "name": "battlefield cross",
+    "description": "A makeshift memorial marker, made of a service rifle - buried in the dirt, or nestled in a pair of boots - with a helmet perched on the stock.  A small way to honor the dead.",
+    "symbol": "(",
+    "color": "dark_gray",
+    "move_cost_mod": 0,
+    "coverage": 20,
+    "required_str": -1,
+    "flags": [ "EASY_DECONSTRUCT", "TRANSPARENT", "MOUNTABLE" ],
+    "deconstruct": {
+      "items": [
+        { "item": "m4_carbine", "count": 1, "charges": 0 },
```

Comment:
`charges: 0` here is intended to make guns spawn without magazines? Or leftover from some copy-paste?

Reply:
Yep, to make them spawn without a mag